charts/bulk-scan-orchestrator/values.aat.template.yaml

```diff
@@ -3,3 +3,10 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   devApplicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+  keyVaults:
+    bulk-scan:
+      resourceGroup: bulk-scan
+      secrets:
```

Comment:
Secret section should include all keys please refer other repos to see how it needs to be configured.